tools/common/config.flow

```diff
@@ -99,6 +99,15 @@ export {
 	// Gather includes from external flow.config's, listed in include_extern as CSV.
 	// When gathering external includes, relative paths are resolved to absolute.
 	fcExternIncludes(include_extern : string) -> [string];
+
+	// Tries to run a program with '-version' option, and output the result.
+	// In case a program doesn't exist on path or any other errors return an empty string.
+	fcProgramVersion(program : string) -> string;
+
+	runGradle(args : [string], currentWorkingDirectory : string, onStdOutLine : (out : string) -> void, onStdErr : (error : string) -> void, onExit : (errocode : int) -> void) -> native;
```

Comment:
This and fcGradleVersion  belongs somewhere in the java/ backend folder.

Reply:
Moved to backends/java/fi2jar_utils.flow